matter_server/server/device_controller.py

```diff
@@ -1236,6 +1239,13 @@ def resubscription_succeeded(
             interval_ceiling = NODE_SUBSCRIPTION_CEILING_BATTERY_POWERED
         else:
             interval_ceiling = NODE_SUBSCRIPTION_CEILING_THREAD
+        if node.attributes.get(ICD_ATTR_LIST_ATTRIBUTE_PATH) is not None:
+            # for ICD devices, the interval floor must be 0 according to the spec,
+            # to prevent additional battery drainage. See Matter core spec, chapter 8.5.2.2.
+            # TODO: revisit this after Matter 1.4 release (as that mighht change this again).
+            interval_floor = 0
```

Comment:
Can we use a variable here too? `NODE_SUBSCRIPTION_FLOOR_ICD` or so?

Reply:
done